package.json

```diff
@@ -18,7 +18,9 @@
     "Richard Chamberlain <richard_chamberlain@uk.ibm.com> (https://github.com/rnchamberlain)"
   ],
   "scripts": {
-    "test": "tap test/test*.js"
+    "test": "npm run lint && npm run tap",
+    "tap": "tap --timeout 10 test/test*.js",
+    "lint": "python cpplint.py **/*.cc **/*.h"
```

Comment:
Sorry, just realized, this should be:

```
   "test": "tap --timeout 10 test/test*.js",
   "pretest": "python cpplint.py **/*.cc **/*.h"
```

And the tap timeout change is unrelated to this PR, shouldn't it be a different PR?